src/gui/MainFrame.cpp

```diff
@@ -542,7 +542,7 @@ void MyMainFrame::OpenProject(wxString project_filename) {
                 button  = "Open";
             }
             else {
-                message = "cisTEM can try to update the format. It is wise to make a backup of the database before trying this.\n\nAttempt to update the project?";
+                message = "cisTEM can try to update the format, which for databases with many classifications and refinements from earlier versions of cisTEM can take a long time. It is wise to make a backup of the database before trying this.\n\nAttempt to update the project?";
```

Comment:
Would it be very difficult to add a button to actually make the backup before proceeding? Maybe a second button that is either "backup" or "update and backup"

Reply:
This is a good idea; I will need to create a custom dialog for it, but that's not complicated.

Reply:
Great! As long as you think the roi is there please do

Reply:
Alright, it took a bit of reorganizing the logic of the update process, and creation of a custom of a dialog, but I've got an implementation that will create a backup of the .db file before proceeding to update the database - please take a look whenever.